### ISSUE TITLE
Wide Review Comment 2017: Content signalling

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/webvtt#368]
Copy/paste from https://lists.w3.org/Archives/Public/public-tt/2017Sep/0080.html - raising as an issue for tracking/disposition purposes.

WebVTT is able to carry data intended for different purposes – captions, chapters, metadata etc. However the lack of signalling within documents creates an asset management and validation problem. If we have 3 WebVTT files associated with a single video asset, how do we know which is which? How would we validate them? The same issue arises with languages: it is impossible to signal the primary language for a WebVTT file within the file; requiring it to be set by the HTML srclang attribute is, frankly, completely useless within the scope of the WebVTT specification. How can an HTML page be generated to specify it correctly, and how can the value be validated for any given WebVTT file unless the language identifier is present within it? What if the WebVTT file is intended for use without being embedded in an HTML document?

The three types of WebVTT file indicated in §4.6 appear to be indistinguishable by inspecting the file.

**Comment by dwsinger** [comment]
seems to be mostly a duplicate of #363. Indeed, usage intent is expected to be signaled externally (and HTML is not the only environment that can do this; DASH also can). I have a personal preference for self-describing files, but given environments that signal language, intended use, etc., one can then end up with conflicts, so we decided long ago not to have intrinsic marking in addition. This is an old architectural question and I think unsuitable for resolution at the end of a second wide-review call.

**Comment by nigelmegitt** [comment]
Agreed, large overlap with #363. Glad we share that preference, but for our use it's more than a preference - in operational situations where a support team needs to trace an issue, inspecting the files is a reasonable thing to do. The conflicts that are seen as a potential problem are a potential problem, but cannot be identified now without information in the file!

I generally prefer not to create situations where there can be an unresolvable internal inconsistency within an asset like a caption file, but where there are multiple layers or assets referencing other assets, the possibility of an inconsistency is unavoidable so it's vital to include the tools to identify and fix it when it arises.

This may be an old architectural question, but it hasn't been diminished by longevity.

**Comment by dwsinger** [comment]
Similar comment; if in your production workflow you need intrinsic information, you can have a comment convention for such for VTT files that are being made for use within the BBC, that include language, timecode, asset-ID, responsible captioner name, fallback lawyer name, whatever...

**Comment by nigelmegitt** [comment]
Likewise, as per https://github.com/w3c/webvtt/issues/369#issuecomment-333208475

**Comment by silviapfeiffer** [comment]
As per https://github.com/w3c/webvtt/issues/363#issuecomment-333414064 , this can wait until V2 of WebVTT.

**Comment by silviapfeiffer** [comment]
See https://www.w3.org/Bugs/Public/show_bug.cgi?id=18657

**Comment by silviapfeiffer** [comment]
FYI: This issue was closed because it had been resolved previously in the CG for the v1 WebVTT spec and it was determined to delay this for v2, as documented above.

Therefore, following https://www.w3.org/wiki/TimedText/WebVTT_Wide_Review#Guidelines_for_Status_of_comments , the CG processing resolution is:

2.7- Done but comment Rejected

And the classification of the issue is:
c- substantive

**Comment by dwsinger** [comment]
Adding -CG-rejected as the proposed resolution there (and once the CG and/or WG agrees, I'll remove the -open labels).

**Comment by silviapfeiffer** [comment]
In https://github.com/w3c/webvtt/pull/394 there's a spec update, so hopefully we can get agreement to resolve this and close the issue.

**Comment by dwsinger** [comment]
#394 approved...

**Comment by nigelmegitt** [comment]
I don't think the classification of "-CG-rejected" is appropriate here since evidently the point is accepted but the desire is to defer it to a future version. Having said that, I can't identify an obvious better option. Marked as commenter-agreed-partial because we do have partial agreement, and deferring seems reasonable given the stage this spec is at.

**Comment by dwsinger** [comment]
OK, leaving it as "resolved" for this version to do nothing, but marked VTT-next for future
